README.md

```diff
@@ -31,6 +31,11 @@ Usage
 ```
 npm run desktop
 ```
+[Dowload mqtt.zip](https://github.com/asyncapi/simulator/files/13696873/mqtt.zip)
+
```

Comment:
Can you put a title here indicating this is a showcase video? Also place this above the file link.

Reply:
Done !! :tada: